vuepress/docs/v7.1/docs/reference/upgrade.md

```diff
@@ -0,0 +1,74 @@
+# Upgrade Guide E7.1.2 ⇒ E7.1.3
+
+## OLM
```

Comment:
Maybe OpenShift/OLM

Reply:
maybe add 'Guide' too??

OpenShift/OLM Guide  

Down below when it says just Non-OpenShift, it leaves you wondering where you are at

Reply:
sure; i should have mentioned that this isn't meant to be a page - just reworking the steps so the headers were more placeholders/notes. i'll formalize it a bit.